src/Widgets/AppEntry.vala

```diff
@@ -97,6 +97,12 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
         }
 
```

Comment:
We probably should add something like this outside the signal after the checking for the app in `headers` (also makes the `tooltip_text` in the image construction obsolete):
`image.tooltip_text = expander.active ? _("Show less") : _("Show more");`
Also the same line could simplify the if else inside the signal.
Another thing I was wondering is whether having separated translatable strings that should be the same like this might be problematic (it's currently the same with the construction using tooltip_text = _("Show less") and the same being used in the signal)? If so we should probably put it in a method? Or maybe I'm just overthinking :)

Reply:
The comment is a bit unfortunately placed just below is the `expander.toggled.connect` signal handler 😅

Reply:
I think I can even do you one better and use a custom property binding :)